util/rate_limiter_test.cc

```diff
@@ -39,14 +45,14 @@ TEST_F(RateLimiterTest, StartStop) {
 
 TEST_F(RateLimiterTest, GetTotalBytesThrough) {
   std::unique_ptr<RateLimiter> limiter(NewGenericRateLimiter(
-      20 /* rate_bytes_per_sec */, 1000 * 1000 /* refill_period_us */,
+      200 /* rate_bytes_per_sec */, 1000 * 1000 /* refill_period_us */,
```

Comment:
[Note]
As mentioned in the PR summary, increasing the `rate_bytes_per_sec` and `request_byte` increase the test's running time since now `Request()` goes through the "long path" of calling `RefillBytesAndGrantBytes()`. If we really hate this increased time, I can also decrease `refill_period_us` to shorten the test time but it will add a bit of mental calculation of `refill_bytes_per_period` when reading the test :)

Reply:
I think it's fine. One day we can do T100099133 to improve the running time.